db/migrate/058_name_unnamed_redactions.rb

```diff
@@ -0,0 +1,19 @@
+class NameUnnamedRedactions < ActiveRecord::Migration[5.0]
+  def self.up
+    Redaction.find_each do |redaction|
+        if redaction.title.empty?
+            redaction.title = redaction.id.to_s
+            redaction.save!
+        end
+    end
```

Comment:
After you fixed any blank titles you need to use `change_column` to add a "not null" constraint to it.